metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/ExecutionTimeAthenaAggregator.java

```diff
@@ -1,22 +1,38 @@
 package io.dockstore.metricsaggregator.helper;
 
+import io.dockstore.metricsaggregator.MetricsAggregatorAthenaClient;
 import io.dockstore.metricsaggregator.MetricsAggregatorAthenaClient.QueryResultRow;
 import io.dockstore.openapi.client.model.ExecutionTimeMetric;
 import java.util.Optional;
 
 /**
  * Aggregate execution time metrics by calculating the min, average, max, and number of data points using AWS Athena.
- * TODO: Implement the executiontime aggregator. Dockstore records executiontime metrics as an ISO-8601 duration which does not translate well in SQL and Athena. We may need to re-evalute how we record this metric.
  */
-public class ExecutionTimeAthenaAggregator extends AthenaAggregator<ExecutionTimeMetric> {
+public class ExecutionTimeAthenaAggregator extends RunExecutionAthenaAggregator<ExecutionTimeMetric> {
+
+    public ExecutionTimeAthenaAggregator(MetricsAggregatorAthenaClient metricsAggregatorAthenaClient, String tableName) {
+        super(metricsAggregatorAthenaClient, tableName);
+        this.addSelectFields(getStatisticSelectFields());
+    }
 
     @Override
-    public String getMetricColumnName() {
-        return "executiontime";
+    String getMetricColumnName() {
+        return "executiontimeseconds";
```

Comment:
Nit, might be nice to have a "constants" file with this and other column names, so you can see in one place what all the column names are.

Update, I guess these are already [these values](https://github.com/dockstore/dockstore-support/blob/078a17204bd50922fe65d65ae3dde3db620f6a79/metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/RunExecutionAthenaAggregator.java#L36)?

Reply:
> Update, I guess these are already [these values](https://github.com/dockstore/dockstore-support/blob/078a17204bd50922fe65d65ae3dde3db620f6a79/metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/RunExecutionAthenaAggregator.java#L36)?

Oh yes they are, I'll update these usages to the constants